langlib/langlib-test/src/test/resources/test-src/arraylib_test.bal

```diff
@@ -190,7 +190,7 @@ function testLastIndexOf() {
 
 function testReverse() returns [int[], int[]] {
```

Comment:
The function `returns` definition seems wrong with the new changes. Are we sure that this didn't give compilation errors?